unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -76,7 +76,7 @@ public class Config {
      */
     private static File applicationDir;
     private static File unitexToolLogger;
-    
+
```

Comment:
It is advised to avoid (unnecessary) spaces refactoring in a 'fix' PR. Alternatively, you should open two PRs : 1) 'refactor' and 2) 'fix' or check [here](https://stackoverflow.com/a/7149602) to learn how to add only the changes that are not white spaces.

Reply:
### Checklist:
   ...
- [ ] My code does not refactor the surrounding code unless necessary

...

Reply:
Thank you for this help.